### ISSUE TITLE
A11y_SQL Azure Data Studio_Import ExtensionModify Columns_AI4D: The primary key check box in column 3 has no name set.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12981]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)– Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG4.1.2;#Benchmark;#AI4D;#AILimited;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW Right click on/ open the context menu for a database and select Data-tier application wizard from the context menu
4. Add data in specify input file screen and tab to "next" button and press enter.
5. Press enter on next button.
6. Launch accessibility insights for desktop and hover over checkbox in column 3 row 1

**Actual:**
On hovering over the checkbox in column 3 row 1 there is no name set for it.
Same issue for the controls in row 1 column 4 and row 2 column 1,2,3,4

**Expected:**
The name should be set for all controls according to its use.

**User Impact:**
The screen reader users will not get the right reference for that control on what it is used for.

**Attachment:**
![#12981_A11y_SQL Azure Data Studio_Import Extension - Import ExtensionModify Columns_AI4D](https://user-images.githubusercontent.com/24580570/96408081-62186d00-1200-11eb-9e1a-febf2c5f5309.png)

**Comment by fsteffi** [comment]
hi @alanrenmsft has the item "Data-tier application wizard " been removed

GithubTags:#Regressed:02-20-23;

**Comment by fsteffi** [comment]
issue not fixed on 
Version: 1.42.0-insider
Commit: 872095ed849537d6c96373eedcd187b3f707c460
Date: 2023-02-23T06:29:37.626Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300
[12981.webm](https://user-images.githubusercontent.com/24580570/220863202-fc7ea82a-d269-4834-bcd2-a8cbd522d5e8.webm)

**Comment by aasimkhan30** [comment]
![image](https://user-images.githubusercontent.com/6816294/221071990-83d3f23d-e5cf-4c7d-ba27-cd830423c329.png)
![image](https://user-images.githubusercontent.com/6816294/221072016-f2e31436-be3b-424b-9fc0-75e50da157f8.png)


@fsteffi , I can see the name attribute on the checkbox. Can you please re-verify this issue.

**Comment by fsteffi** [comment]
@aasimkhan30 the names are not set for the checkcbox in row 2

![image](https://user-images.githubusercontent.com/24580570/223423879-e7574c9a-ba56-4acb-b047-0c82efa78e79.png)

please reopen this bug

**Comment by aasimkhan30** [comment]
@fsteffi , that's a different bug. The bug is only referring to the header checkboxes. Can you please file a new bug with a new SLA and close this issue?

**Comment by Roopesh-T** [comment]
Closing this bug as logged a new bug. https://github.com/microsoft/azuredatastudio/issues/22372 

**GitHub Tags** : #closed;
